ros_gz_sim/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
```

Comment:
you don't need to include this dependency, with rclcpp should be enough, you are not adding any component

Reply:
If I don't add it, the code doesn't compile. Maybe a dependency isn't exporting it correctly.

Reply:
I think we determined that ros_gz_bridge isn't correctly exporting the dependency.

Reply:
Removed!